config/recipe.yml

```diff
@@ -50,13 +50,21 @@ modules:
       - Celluloid: io.github.celluloid_player.Celluloid
       - Krita: org.kde.krita
 
-  - type: systemd
-    system:
-      enabled:
-        - power-profiles-daemon
-        - gdm
-      disabled:
-        - rpm-ostree-countme.timer
+  # - type: systemd
+  #  system:
+  #    enabled:
+  #      - example.service
+  #      - example.timer
+  #    disabled:
+  #      - example.service
+  #      - example.timer
```

Comment:
Quickly looking at this today, we can’t really have commented-out configuration for _every module_ when there’s no benefit for including default config.

Reply:
sounds good, it was mainly to demonstrate what the module looks like since I don't have a README for it yet. I'll probably remove it along with the commend above the files module, moving both of them into the docs